src/main/java/org/folio/circulation/CirculationVerticle.java

```diff
@@ -74,7 +77,19 @@ public void start(Promise<Void> startFuture) {
     Router router = Router.router(vertx);
 
     // bump up the connection pool size from the default value of 5
-    final HttpClient client = vertx.createHttpClient(new HttpClientOptions().setMaxPoolSize(100));
+    String maxPoolSizeEnv = getenv().getOrDefault("HTTP_MAXPOOLSIZE", String.valueOf(
+      HTTP_MAXPOOLSIZE_DEFAULT));
+    int maxPoolSize = HTTP_MAXPOOLSIZE_DEFAULT;
+
+    try {
+      maxPoolSize = Integer.parseInt(maxPoolSizeEnv);
```

Comment:
Can we extract it into a method? Maybe even into a utility class, e.g. in existing class `CommonUtils`?

Reply:
done